src/vs/workbench/contrib/testing/common/testResult.ts

```diff
@@ -379,6 +379,24 @@ export class LiveTestResult implements ITestResult {
 		}
 
 		const index = this.mustGetTaskIndex(taskId);
+
+		const terminalStates: { [key in TestResultState]?: number } = {
+			[TestResultState.Passed]: 0,
+			[TestResultState.Skipped]: 1,
+			[TestResultState.Failed]: 2,
+			[TestResultState.Errored]: 3,
+		};
```

Comment:
I think this overall looks good, can you move this object into `testingStates.ts`?

Reply:
Done, I force-pushed a new version. I decided to rename it to `terminalStatePriorities`, that's a little more precise.